SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005752	Attack enemies in the area by exploding the gunpowder in hand. This technique will also damage the user and might cause combat incapability if the HP is too low.
+SKILL_20151224_005753	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: %{CaptionRatio}#%{nl}HP consumption: #{CaptionRatio2}#
+SKILL_20151224_005754	Creates clones of yourself. Clones will act along with the main body and skills can be used along with the clones. HP of the clones is determined by the user's dexterity.
+SKILL_20151224_005755	Duration: #{CaptionRatio}#second{nl}Clone summon count: #{CaptionRatio2}#
+SKILL_20151224_005756	Counter-attack by tricking the enemy with fake doll created from a log.
+SKILL_20151224_005757	Attack multiple enemies by continuously exploding gunpowder in hand.
+SKILL_20151224_005758	Attack multiple enemies by throwing kunais.
+SKILL_20151224_005759	Dragontooth
+SKILL_20151224_005760	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Rotate your spear using centrifugal force to attack the enemy.
+SKILL_20151224_005761	Serpentine
+SKILL_20151224_005762	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Use your Spear to deal continuous damage to enemies in front of you.
+SKILL_20151224_005763	Gae Bulg
+SKILL_20151224_005764	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Throw your spear forward to attack multiple enemies. The thrown spear must be recovered after skill use.
+SKILL_20151224_005765	Dragon Soar
+SKILL_20151224_005766	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Vault into the air while swinging your spear upward. Enemies hit with the skill are temporarily launched.
+SKILL_20151224_005767	Summon Guild Member
+SKILL_20151224_005768	Summon designated guild member to their current location. Need an approval from the designated guild member. Can't summon guild members in an instance dungeon.
+SKILL_20151224_005769	Guild member summon count: #{CaptionRatio}#
+SKILL_20151224_005770	Warp To Guild Member
+SKILL_20151224_005771	Warp to designated guild member's location. Can't warp to guild members in an instance dungeon.
+SKILL_20151224_005772	Warp To Designated Guild Member's Location
+SKILL_20151224_005773	Build Guild Tower
+SKILL_20151224_005774	Build guild tower in the designated location. Can only be built in a field and can't be built in an instance dungeon.{nl}Level 1: Warp allowed{nl}Level 2: Storage allowed{nl}Level 3: Guild growth allowed{nl}Level 4: Guild event allowed{nl}Level 5: Guild hideout allowed
+SKILL_20151224_005775	Guild Tower Function Improvement
+SKILL_20151224_005776	Reduce Craft Time
+SKILL_20151224_005777	Plant a flag that reduces crafting time of all guild members in the nearby area. Can only be used near the guild tower.
+SKILL_20151224_005778	Decrease craft time: #{CaptionRatio}#% 
+SKILL_20151224_005779	Battle Orders
+SKILL_20151224_005780	Increase the strength of guild members around the guild master. Effect disappears if the guild master is too far away.
```

Comment:
Temporarily increases STR of guild members nearby the guild master. This effect disappears if the guild master is too far away.